bundles/net.tourbook.web/WebContent-rel/tourbook/package.json

```diff
@@ -5,5 +5,5 @@
 	"description":	"MyTourbook web applications.",
 	"homepage":		"http://mytourbook.sourceforge.net",
 	
-	"dojoBuild":	"build-profile.js",
+	"dojoBuild":	"build-profile.js"
```

Comment:
I've no idea what this is fixing but the ant `net.tourbook.web CREATE-Dojo-Bundle.xml` is still running without problems

Reply:
Theoretically, the extra `,` is not allowed and the build was complaining about. Now it should be good. I am also not familiar a lot with Dojo